[2021-08-04 13:29:37,505] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:29:32.937004+00:00 [queued]>
[2021-08-04 13:29:37,662] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:29:32.937004+00:00 [queued]>
[2021-08-04 13:29:37,662] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:29:37,662] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:29:37,663] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:29:37,726] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_B> on 2021-08-04T13:29:32.937004+00:00
[2021-08-04 13:29:37,746] {standard_task_runner.py:52} INFO - Started process 10050 to run task
[2021-08-04 13:29:37,794] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_B', '2021-08-04T13:29:32.937004+00:00', '--job-id', '1509', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpgillg3ea', '--error-file', '/tmp/tmpoan_4uoc']
[2021-08-04 13:29:37,795] {standard_task_runner.py:77} INFO - Job 1509: Subtask training_model_B
[2021-08-04 13:29:37,956] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_B 2021-08-04T13:29:32.937004+00:00 [running]> on host 04479ea62028
[2021-08-04 13:29:38,326] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:29:32.937004+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:29:32.937004+00:00
[2021-08-04 13:29:38,327] {python.py:151} INFO - Done. Returned value was: 8
[2021-08-04 13:29:38,468] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_B, execution_date=20210804T132932, start_date=20210804T132937, end_date=20210804T132938
[2021-08-04 13:29:38,744] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:29:38,788] {local_task_job.py:149} INFO - Task exited with return code 0
